config/cmake/addlibzmq.cmake

```diff
@@ -124,6 +125,17 @@ set(WITH_PERF_TOOL
     OFF
     CACHE INTERNAL ""
 )
+
+set(WITH_LIBSODIUM_STATIC
+    OFF
+    CACHE INTERNAL ""
+)
```

Comment:
Okay. `ENABLE_CURVE` is also off so this shouldn't have the TweetNacl mutex.hpp error that we were seeing before.

Reply:
This was a new option to enable a static libsodium,  it is off by default,  I just didn't want to see it the CMAKE options for HELICS.